src/cpu/aarch64/acl_convolution_utils.cpp

```diff
@@ -156,10 +156,18 @@ status_t acl_init_conf(acl_conv_conf_t &acp, memory_desc_t &src_md,
     const auto acl_layout = is_nspc ? arm_compute::DataLayout::NHWC
                                     : arm_compute::DataLayout::NCHW;
 
-    auto acl_src_data_t = acl_utils::get_acl_data_t(src_d.data_type());
-    auto acl_wei_data_t = acl_utils::get_acl_data_t(wei_d.data_type());
-    auto acl_dst_data_t = acl_utils::get_acl_data_t(dst_d.data_type());
-    auto acl_bia_data_t = acl_utils::get_acl_data_t(bia_d.data_type());
+    // For convolutions, int8 datatypes imply quantized types in ACL
+    acp.is_int8 = utils::one_of(src_d.data_type(), s8, u8)
+            && wei_d.data_type() == s8;
```

Comment:
Minor: I tend to ask to leave `wei_d.data_type() == s8` since this is the only characteristic of quantized operation. Activations may be of `bf16` or `f32` types but this still will be a quantized operation. I'm fine to leave as is if this reflects the state of ACL support and the desire is to keep it this way. Thank you.

Reply:
Hi @dzarukin, the code as is still includes the `wei_d.data_type() == s8` check, right?

Otherwise I apologise as I'm not sure I understand what you are suggesting? Thanks!

Reply:
Hi @diaena, I'm suggesting to drop a check for source data type since it may be any (at least, API allows any of supported data types). But, as I mentioned, if there's no desire to support configurations like bf16s8bf16 of bf16s8x8, it's fine to keep it and resolve the discussion. Thank you.

Reply:
Oh I understand now! Thanks for clarifying. I will leave it as it is because it does indeed capture a limitation of Compute Library where src and weights have to have matching datatypes.